libs/sr/ui/src/Character/CharacterCard.tsx

```diff
@@ -0,0 +1,28 @@
+import { CardThemed } from '@genshin-optimizer/common/ui'
+import { ICachedSroCharacter } from '@genshin-optimizer/sr/db'
+import { CardContent, Stack } from '@mui/material'
+
+export function CharacterCard({
+  character,
+}: {
+  character: ICachedSroCharacter
+}) {
+  if (!character) {
```

Comment:
Proper typing should prevent this from happening, there is no need to write defensive code here against null `character`.